workspaces/tech-insights/.changeset/eighty-brooms-repeat.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/plugin-tech-insights-backend': patch
```

Comment:
This looks to be a breaking change with a new mandatory dependency. Should this be a major bump?

Reply:
I wasn't sure. Should I change it to major bump?